docs/api/distance.md

```diff
@@ -0,0 +1,13 @@
+# Distance functions
+
+## h3Distance
+
+```
+int h3Distance(H3Index origin, H3Index h3);
+```
+
+Returns the distance in grid cells between the two indexes.
+
+Returns a negative number if finding the distance failed. Finding the distance can fail because the two
```

Comment:
Is it worth outlining the meaning of the different error codes here?

Reply:
`h3Distance` itself only returns -1 on any kind of error right now, it doesn't preserve the more specific error code from `h3ToIjk`